src/views/CHConfigEditor.tsx

```diff
@@ -1,31 +1,20 @@
-import React, { ChangeEvent, useState } from 'react';
+import React, { ChangeEvent, useMemo, useState } from 'react';
 import {
   DataSourcePluginOptionsEditorProps,
   onUpdateDatasourceJsonDataOption,
   onUpdateDatasourceSecureJsonDataOption,
 } from '@grafana/data';
-import {
-  Button,
-  InlineFormLabel,
-  LegacyForms,
-  RadioButtonGroup,
-  useTheme,
-  Switch,
-  InlineFieldRow,
-  InlineField,
-  Input,
-} from '@grafana/ui';
+import { RadioButtonGroup, Switch, Input, Field, SecretInput, Button, InlineField, InlineFieldRow } from '@grafana/ui';
 import { CertificationKey } from '../components/ui/CertificationKey';
 import { Components } from './../selectors';
-import { config } from '@grafana/runtime';
 import { CHConfig, CHCustomSetting, CHSecureConfig, Protocol } from './../types';
 import { gte } from 'semver';
+import { ConfigSection, ConfigSubSection, DataSourceDescription } from '@grafana/experimental';
```

Comment:
Important to call out that these will end up being accepted into either `@grafana/ui` or another similar package that isn't experimental.

Reply:
Would you prefer a comment in the code or on the PR description?

Reply:
Sorry, I was fine just calling it out here :sunglasses: